doc/kubernetes/modules/ROOT/pages/running/index.adoc

```diff
@@ -5,5 +5,6 @@
 It summarizes the logic which is condensed in the Helm charts and scripts in this project to make it accessible as independent knowledge to adapt it to other environments.
 
 * xref::running/keycloak-deployment.adoc[]
+* xref::running/keycloak-with-external-infinispan-deployment.adoc[]
```

Comment:
can you add `running/keycloak-with-external-infinispan-deployment.adoc` to `nav.adoc`?

Reply:
Oops, forgot that. Fixed it in my next commit